agent/src/com/cloud/agent/Agent.java

```diff
@@ -227,6 +227,7 @@ public void start() {
         try {
             _connection.start();
         } catch (final NioConnectionException e) {
+            s_logger.warn("NIO Connection Exception  " + e);
```

Comment:
Since it is unlikely that these two lines will appear in the log sequentially, why aren't these two log messages combined into one ``WARN`` message?